docs/sphinx/source/contributing.rst

```diff
@@ -69,11 +69,13 @@ Requests <https://help.github.com/articles/using-pull-requests/>`_.
 
 We strongly recommend using virtual environments for development.
 Virtual environments make it trivial to switch between different
```

Comment:
```suggestion
Virtual environments make it easy to switch between different
```
It honestly isn't trivial for new users

Reply:
True (wasn't my wording btw)
how about "easier" rather than "easy" though?